src/ui/FavoriteTextCmp.js

```diff
@@ -1,7 +1,16 @@
+import sanitize from '../util/sanitize'
 import fs from './FavoriteStyle';
 import {onMarkdownEditorKeyDown} from '../util/dom';
 
 export default function({ layout, i18n }) {
+    const getValue = function() {
+        var t = this,
+            value = sanitize(t.rawToValue(t.processRawValue(t.getRawValue())));
+
+        t.value = value;
```

Comment:
is this necessary?

Reply:
I override the getValue method so that you always get the sanitized version from these textfields. Not necessary, just to make sure we won't forget to sanitize later.

Reply:
Or do you mean line 10? I just copied the getValue method from the Ext source and added `sanitize` on line 8. Could potentially reference the source function instead and wrap it, but not much difference.

Reply:
@edoardo